src/notification-manager.coffee

```diff
@@ -46,8 +56,18 @@ class NotificationManager
   #
   # * `message` A {String} message
   # * `options` (optional) An options {Object} with the following keys:
-  #    * `detail` (optional) A {String} with additional details about the
-  #      notification.
+  #    * `buttons` (optional) An {Array} of {Object} where each {Object} has the
+  #      following options:
+  #      * `className` (optional) {String} additional class name add to the
+  #        button. It will already receive `btn btn-info`.
+  #      * `onDidClick` (optional) {Function} callback to call when the button
+  #        has been clicked. The context will be set to the
+  #        {NotificationElement} instance.
+  #      * `text` {String} inner text for the button
+  #    * `description` (optional) A {String} that will be rendered as Markdown
+  #      to describe the notification.
+  #    * `detail` (optional) A preformatted {String} that will be rendered as
```

Comment:
That's a good point. 👍 I wonder if, however, this is just an implementation detail of the `notifications` package rather than being intrinsic to the `detail` field of the `Notification` object.

That said, I can see how it could cause some misunderstandings, so what about something like:

```
A plain-text {String} containing additional details about the notification. By default, this **will** preserve newlines and whitespace when it is rendered.
```

Similarly, we can document the `description` field like:

```
A Markdown {String} containing a longer description about the notification. By default, this **will not** preserve newlines and whitespace when it is rendered.
```